42manito/src/hooks/Mentor/MentorModal.tsx

```diff
@@ -20,7 +27,25 @@ export const useMentorModal = () => {
         htmlElement.style.overflowY = "unset";
       }
     }
-  }, [currMentorState.openMentorModal]);
+  }, [currMentorState.openMentorModal, dispatch]);
```

Comment:
dispatch 의존성 제거해도 되지 않나요?
```suggestion
  }, [currMentorState.openMentorModal]);
```

Reply:
네 돼요 수정하다가 코파일럿 때문에 들어갔나봐요ㅜㅜ